lightkurve/mast.py

```diff
@@ -61,86 +69,242 @@ def search_kepler_products(target):
+        suffix = "{} Short".format(filetype)
+    elif cadence in ['any', 'both']:
+        suffix = "{}".format(filetype)
     else:
-        suffix = "lpd-targ"
-    mask = np.array([suffix in fn for fn in products['productFilename']])
-    # Identify the campaign or quarter by the description.
-    quarter_or_campaign = campaign if campaign is not None else quarter
-    if quarter_or_campaign is not None:
-        mask &= np.array([desc.lower().endswith('q{}'.format(quarter_or_campaign)) or
-                          desc.lower().endswith('c{:02d}'.format(quarter_or_campaign))
-                          for desc in products['description']])
-    return products[mask]
+        suffix = "{} Long".format(filetype)
+    if len(products) > 0:
+        # Identify the campaign or quarter by the description.
+        if qoc[0] is not None:
+            mask = np.zeros(np.shape(products)[0], dtype=bool)
+            for q in qoc:
+                if q is not None:
+                    mask |= np.array([desc.lower().endswith('q{}'.format(q)) or
+                                      desc.lower().endswith('c{:02d}'.format(q))
+                                      for desc in products['description']])
+        else:
+            mask = np.ones(np.shape(products)[0], dtype=bool)
+        mask &= np.array([desc.lower().endswith('fits') or
+                          desc.lower().endswith('fits.gz')
+                          for desc in products['dataURI']])
+        mask &= np.array([suffix in desc for desc in products['description']])
+        products = products[mask]
 
+    #Add the quarter or campaign numbers
+    qoc = np.asarray([p.split(' - ')[-1][1:]
+                      for p in products['description']], dtype=int)
+    products['qoc'] = qoc
+     #Add the dates of each short cadence observation to the product table.
+    dates = [p.split('/')[-1].split('-')[1].split('_')[0]
+             for p in products['dataURI']]
+    for idx, d in enumerate(dates):
+        try:
+            dates[idx] = float(d)
+        except:
+            dates[idx] = 0
+    products['dates'] = np.asarray(dates)
+    return products
 
-def search_kepler_lightcurve_products(target, cadence='long', quarter=None,
-                                      campaign=None):
-    """Returns a table of Kepler or K2 lightcurve files for a given target.
+def get_kepler_products(target, filetype='Target Pixel', cadence='long',
+                        quarter=None, month=None, campaign=None, radius=1.,
+                        targetlimit=1, verbose=True, **kwargs):
+    """Fetch files from from the Kepler/K2 data archive at MAST.
 
-    This only returns products produced by the official Kepler pipeline,
-    which is not necessarily the best choice for every use case.
+    Raises an `ArchiveError` if a unique TPF cannot be found.  For example,
+    this is the case if a target was observed in multiple Quarters and the
+    quarter parameter is unspecified.
```

Comment:
We'll have to update this description to explain that multiple files can be returned now.